src/datasets/other/activity-stream/reference.md

```diff
@@ -0,0 +1,131 @@
+
+* **Inconsistent fields**: In some tables, the same field can have different meanings for different records. 
+
+	For example, in the `assa_router_events_daily` table, the `impression_id` field corresponds to the standard Telemetry `client_id` field for [Snippets impressions](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_events.md#snippets-impression), [CFR impressions for pre-release and shield experiments](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_events.md#cfr-impression-for-all-the-prerelease-channels-and-shield-experiment), and [onboarding impressions](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_events.md#onboarding-impression). However, for [CFR impressions for release](https://github.com/mozilla/activity-stream/blob/master/docs/v2-system-addon/data_events.md#cfr-impression-for-the-release-channel), this field is a separate, impression identifier. 
+
+* **Passing Experiment Tags**: If a profile is enrolled in a Normandy experiment, the experiment slug for that profile is only passed to the Activity Stream data if it contains the string "activity-stream". 
+
+	In other words, Activity Stream will not tag data as belonging to an experiment if it is missing "activity-stream" in the slug, even if it is indeed enrolled in an experiment. 
+
+* **Data field formats**: The format for some of the data that is shared with standard Telemetry can differ. 
+
+	For example, experiment slugs in standard Telemetry is formatted as an array of maps (one for each experiment the profile is enrolled in) 
+
+	`[{'experiment1_name':'branch_name'}, {'experiment2_name':'branch_name'}]`
+
+	However, in the Activity Stream, experiment slugs are reported in a string, using `;` as a separater between experiments and `:` as a separator between experiment name and branch name. 
+
+	`'experiment1_name:branch_name;experiment2_name:branch_name'`
+
+* **Null handling**: Some fields in the Activity Stream data encode nulls with a `'N/A'` string or a `-1` value. 
+
+* **Changes in ping behaviors**: These pings continue to undergo development and the behavior as well as possible values for a given ping seem to change over time. For example, older versions of the event pings for clicking on a Topsite do not seem to report `card_types` and `icon_types`, while newer versions do. Caution is advised. 
+
+* **Pocket data**: Data related to Pocket interaction and usage in the `about:home` and `about:newtab` pages get sent to Pocket via this data collection and pipeline. However, due to privacy reasons, that data is sanitized and `client_id` is randomized. So while it is possible to ask, "how many Topsites and Highlights did a given profile click on in a given day?", we cannot answer that question for Pocket tiles. 
+
+
+## Examples
+
+#### Sessions per `client_id`
+
+Note: only includes `client_ids` that completed an Activity Stream session that day. 
```

Comment:
I think the wording here might be confusing. 

I'm differentiating between activity stream sessions and firefox sessions. 

if they have any activity stream session (meaning a homepage/newtab that was activity stream was loaded, not necessarily that any user action had to be taken during that activity stream session), they'll be included. 

the statement "that concluded a session that day, which contained Activity Stream activity" isn't strictly correct, since a user could have started an activity stream session, completed an activity stream session (navigate out or close that window/tab), and still not complete the firefox session.